packages/yum/test.sh

```diff
@@ -153,13 +160,76 @@ function mroonga_is_registered() {
   sudo systemctl stop ${service_name}
 }
 
+function mroonga_is_registered_for_mysql_community_minimal() {
+  sudo mkdir -p /var/lib/mysql /var/run/mysqld; \
+  sudo chown mysql:mysql /var/lib/mysql /var/run/mysqld; \
+  sudo chmod 1777 /var/lib/mysql /var/run/mysqld
+
+  auto_generated_password=$(mysqld --initialize |& awk 'END{print $NF}')
+  mysql="mysql -u root -p${auto_generated_password}"
+  mysqld &> /dev/null &
```

Comment:
```suggestion
  mysqld &
```

Should we stop the `mysqld` process after we finish this test?

Reply:
I think we should not stop the mysqld process after we finish this test.